### DISCUSSION TITLE
Where to find singers

### DISCUSSION THREAD

**Discussion by home-gihub** [stakira/OpenUtau#775] [Q&A]
i have no idea where to find them. any websites people would like to recommend?
P.S. i don't have discord

**Comment by pseudo-producer** [comment]
It is a situation that depends on a multitude of factors. Voicebanks are not linked to Open Utau, as they are in Synth V, as they are free and anyone can develop their own. The best you can do is research about voicebanks for UTAU, as Open Utau is an open source version of UTAU. The best known banks are Japanese, such as Kasane Teto and Namine Ritsu, but they exist in other languages. Search the Utau Wiki because it will be a good start!

**Reply by pseudo-producer** [reply]
Hello! I'm glad to have helped! As OpenUtau is based or inspired by UTAU (2008) I imagine that the creation is the same for both software, in addition to the voice banks being compatible for both software.

**Reply by pseudo-producer** [reply]
In short, just search for "how to create UTAU voice banks" or "voicebanks for UTAU", if you just want to test an existing voice bank, which I imagine will work.

**Reply by pseudo-producer** [reply]
Regarding the issue of the document/instruction, right here in the project's GIT HUB there is the "WIKI" tab that breaks down the functionality of the software.
